### DISCUSSION TITLE
How would I display a translation in a o2m relationship?

### DISCUSSION THREAD

**Discussion by LevinGermann** [directus/directus#9524] [Archive]
I have this setup:

**Location** <_o2m_> **Quiz** <_o2m_> **Answer**

An answer collection contains a flag if it is correct and a translations object for the answer possibilities.
With the new display type, I can preview such answers nicely if I just look at the Answers collection.

<img width="402" alt="image" src="https://user-images.githubusercontent.com/9347206/140521845-9d15807d-74ac-4f39-a76b-59f636f2be53.png">

How would I achieve now in the **Quiz** collection, which references multiple Answer-items, that I can make use of a translation preview including other fields from the collection?
Using the Display Template doesn't seem to work.

<img width="620" alt="image" src="https://user-images.githubusercontent.com/9347206/140522271-314ac3e0-78bf-4030-86df-276996ec4bac.png">

<img width="247" alt="image" src="https://user-images.githubusercontent.com/9347206/140522307-241b5052-805e-4d5e-9db8-84c1894eaa06.png">

Any help is greatly appreciated.
Cheers!

**Comment by christianrr** [comment]
Since "Translations" in the display template is an array of multiple items, the display template would have to know which item to show when displaying the value of "Translations".

I see several possible fixes:

- Either the display template catches up the "preview translations" display settings of the translations field (however would only work for translations), 

or more general:
- always display the first item in O2M or M2M relationships (e.g. _answer Red & Gold_),
- show all values comma separated in O2M or M2M relationships (e.g. _answer Red & Gold, Rot & Gold_ -> potentially long),

or even more generic:
- extend display template to allow for picking something like Translations[1].Title (e.g. _answer Rot & Gold_)

**Reply by LevinGermann** [reply]
Thanks for your reply!
Regarding your first point, that's how I would expect it to work for translations (because of the "Preview Translation" display template). However, it seems like this is not possible (not sure if just not implemented).
I opened an issue here: https://github.com/directus/directus/issues/9751
